src/backend/common/Logger.cpp

```diff
@@ -51,10 +51,11 @@ loggerFactory(string name) {
 
 string bytesToString(size_t bytes) {
     static array<const char *, 5> units{"B", "KB", "MB", "GB", "TB"};
-    int count = 0;
+    unsigned long long count = 0;
```

Comment:
Why not size_t ?

Reply:
Yeah, size_t would be better. I am not sure what I was thinking. Fixed.